Solutions/Lookout/Data Connectors/Lookout_API_FunctionApp.json

```diff
@@ -0,0 +1,118 @@
+				"providerDisplayName": "Keys",
+				"scope": "Workspace",
+				"requiredPermissions": {
+					"action": true
+				}
+			}
+		],
+		"customs": [{
+				"name": "Microsoft.Web/sites permissions",
+				"description": "Read and write permissions to Azure Functions to create a Function App is required. [See the documentation to learn more about Azure Functions](https://docs.microsoft.com/azure/azure-functions/)."
+			},
+			{
+				"name": "REST API Credentials/permissions",
+				"description": "**EnterpriseName** & **ApiKey** are required for REST API. [See the documentation to learn more about API](https://mtp.lookout.com). Check all [requirements and follow  the instructions](https://mtp.lookout.com) for obtaining credentials."
+			}
+		]
+	},
+	"instructionSteps": [{
+			"title": "",
+			"description": ">**NOTE:** This connector uses Azure Functions to connect to the Lookout REST API to pull its events into Azure Sentinel. This might result in additional data ingestion costs. Check the [Azure Functions pricing page](https://azure.microsoft.com/pricing/details/functions/) for details."
+		},
+		{
+			"title": "",
+			"description": "**STEP 1 - Configuration steps for the Lookout API**\n\n [Follow the instructions](https://mtp.lookout.com) to obtain the credentials. \n"
+		},
+		{
+			"title": "",
+			"description": "**STEP 2 - Choose ONE from the following two deployment options to deploy the connector and the associated Azure Function**\n\n>**IMPORTANT:** Before deploying the Lookout data connector, have the Workspace ID and Workspace Primary Key (can be copied from the following).",
+			"instructions": [{
+					"parameters": {
+						"fillWith": [
+							"WorkspaceId"
+						],
+						"label": "Workspace ID"
+					},
+					"type": "CopyableLabel"
+				},
+				{
+					"parameters": {
+						"fillWith": [
+							"PrimaryKey"
+						],
+						"label": "Primary Key"
+					},
+					"type": "CopyableLabel"
+				}
+			]
+		},
+		{
+			"title": "Option 1 - Azure Resource Manager (ARM) Template",
+			"description": "Use this method for automated deployment of the Lookout data connector using an ARM Tempate.\n\n1. Click the **Deploy to Azure** button below. \n\n\t[![Deploy To Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fgate6%2FAzure-Sentinel%2Fmaster%2FSolutions%2FLookout%2FData%2520Connectors%2Fazuredeploy_Connector_LookoutAPI_AzureFunction.json)\n2. Select the preferred **Subscription**, **Resource Group** and **Location**. \n> **NOTE:** Within the same resource group, you can't mix Windows and Linux apps in the same region. Select existing resource group without Windows apps in it or create new resource group.\n3. Enter the **EnterpriseName** & **ApiKey** and deploy. \n4. Mark the checkbox labeled **I agree to the terms and conditions stated above**. \n5. Click **Purchase** to deploy."
```

Comment:
please update short url link for azure deploy as follows:
[https://aka.ms/sentinel-lookoutapi-azuredeploy](https://aka.ms/sentinel-lookoutapi-azuredeploy)

Reply:
Done